src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CaseDataController.java

```diff
@@ -65,6 +65,23 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> setConsentedDefaultV
         return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder().data(callbackRequest.getCaseDetails().getData()).build());
     }
 
+    @PostMapping(path = "/consented/set-amend-defaults", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Remove this method as it is not needed now since we have the null check in the setOrganisationPolicy method

Reply:
OK Ruban, will do thanks